CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (C) 2024 Renesas Electronics Corporation.
+# Copyright (C) 2024 EPAM Systems, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cmake_minimum_required(VERSION 3.19)
+
+# ##############################################################################
+# Generate gRPC stubs
+# ##############################################################################
+
+project(aos_core_api)
+
+# ##############################################################################
+# Options
+# ##############################################################################
+
+option(BUILD_AOS_PROTOCOL_LIB "build aos protocol lib" OFF)
```

Comment:
Let's have a uniform name:

WITH_AOS_PROTOCOL
WITH_IAM_API
WITH_SM_API

Should be some fatal id no options are selected